src/cswinrt/strings/additions/Windows.Foundation/TaskToAsyncOperationWithProgressAdapter.cs

```diff
@@ -15,7 +15,7 @@ namespace System.Threading.Tasks
 #if NET
     [global::System.Runtime.Versioning.SupportedOSPlatform("windows10.0.10240.0")]
 #endif
-    internal sealed class TaskToAsyncOperationWithProgressAdapter<TResult, TProgress>
+    internal sealed partial class TaskToAsyncOperationWithProgressAdapter<TResult, TProgress>
```

Comment:
How did you spot all of these missing `partial`? If this is required to let the generator add the necessary code to make things AOT friendly, should we consider having an analyzer (or the generator itself) spot such cases (ie. types matching some criteria, like being async operations or something) and emit a diagnostic if they're missing `partial`? So that you'd be warned rather than just have the generator silently skip processing the type. What do you think?

Reply:
I do have plans on adding diagnostics to help catch these and warn folks.  Right now, getting things functionally working has been the priority, soon I will start adding stuff like diagnostics.

Reply:
Cool, just figured I'd ask! We can just ignore this for now then, sounds good! 🙂